src/sdk/model/vapp/__json__/vapp-network-initialization-params-request-json.ts

```diff
@@ -0,0 +1,16 @@
+import { IpRangeJson } from '../../common/ip-range/__json__/ip-range-json';
+
+export interface VappNetworkInitializationParamsRequestJson {
```

Comment:
is this actually used anywhere now?

Reply:
indeed you're right. I can get rid of the old methods. I'll change this.